axiom/tests/.gitignore

```diff
@@ -95,7 +95,6 @@ test_segment
 test_segment_children
 test_segment_datastore
 test_segment_external
-test_segment_message
```

Comment:
I think this has been removed by a mistake during rebase.

Reply:
Actually I added it because it was missing then changed my mind and added it to this PR instead:
https://github.com/newrelic/newrelic-php-agent/pull/1026

Reply:
I had it in this PR, removed it, put it in https://github.com/newrelic/newrelic-php-agent/pull/1026 and then rebased.  Weird it didn't complain about a conflict - I'll look into it.